build.gradle

```diff
@@ -29,6 +29,7 @@ ext {
 }
 
 repositories {
+	mavenLocal()
```

Comment:
Since you are now fine with your changes locally, you can now undo this

Reply:
Oops! Done now